lib/mongoid/document.rb

```diff
@@ -102,7 +102,7 @@ def identity
     #
     # @return [ Document ] A new document.
     def initialize(attrs = nil, &block)
-      construct_document(attrs, execute_callbacks: true, &block)
+      construct_document(attrs, execute_callbacks: Threaded.execute_callbacks?, &block)
```

Comment:
Since the method uses this as the default value, you can just omit the argument:
```suggestion
      construct_document(attrs, &block)
```

Reply:
Good catch! Thanks 👍